Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM golang:1.11-alpine3.9 as builder
-RUN apk update &&  apk add gcc musl-dev libc6-compat git make xz-dev
+FROM golang:1.11 as builder
+RUN apt-get update && apt-get install -y gcc gcc-mingw-w64 gcc-multilib git musl-dev libc6 liblzma-dev make
 RUN mkdir -p /go/src/github.com/mendersoftware/mender-artifact
 WORKDIR /go/src/github.com/mendersoftware/mender-artifact
+RUN go get 'golang.org/x/sys/windows'
 ADD ./ .
-RUN make build
+RUN make buildNatives
 
 FROM alpine:3.9
 RUN apk add xz-dev
-COPY --from=builder /go/src/github.com/mendersoftware/mender-artifact/mender-artifact /go/bin/mender-artifact
-ENTRYPOINT [ "/go/bin/mender-artifact" ]
+COPY --from=builder /go/src/github.com/mendersoftware/mender-artifact/ /go/bin/
+ENTRYPOINT [ "/go/bin/mender-artifact-linux" ]
```

Comment:
Are they called this on all three platforms?

Reply:
the entrypoint is only a placeholder here and will be overridden during binary extraction, as the image will contain all builds (so this probably shouldn't be widely used)

but from all 3 binaries, the linux on would be the one able to be run in this container...

apart from that the name always is "mender-artifact-{linux|mac|windows.exe}"